activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+*   `to_sentence` returned a frozen string after `frozen_string_literal: true` got introduced in #29728.
```

Comment:
I don't think we need to mention the cause of the problem here, just the new behaviour. Perhaps simply:

> `Array#to_sentence` no longer returns a frozen string.

Reply:
Thanks @eugeneius for the review. I've addressed your concern. I like that it explicitly mentions `Array` now )